pkg/logql/marshal/query.go

```diff
@@ -3,6 +3,7 @@ package marshal
 import (
 	"fmt"
 
+	lokimodel "github.com/grafana/loki/model"
```

Comment:
@owen-d Hey, In this package, we used other variable also from `github.com/prometheus/common/model`. What should we do in this case ?

Reply:
@owen-d any update?